test/test_compressed_support.py

```diff
@@ -0,0 +1,97 @@
+#-------------------------------------------------------------------------------
+# elftools tests
+#
+# Pierre-Marie de Rodat (pmderodat@kawie.fr)
+# This code is in the public domain
```

Comment:
I just noticed that the repository embeds a pre-built readelf program. Unfortunately, it seems too old to handle compressed sections:

```shell
$ ../external_tools/readelf --version|head -n1
GNU readelf (GNU Binutils) 2.23.52.20130918

$ ../external_tools/readelf -wi compressed_64.o.elf 
readelf: Warning: skipping invalid relocation offset 0x1af in section .debug_info
readelf: Warning: skipping invalid relocation offset 0x1bc in section .debug_info
[…]
Contents of the .debug_info section:

  Compilation Unit @ offset 0x0:
   Length:        0x1 (32-bit)
   Version:       0
   Abbrev Offset: 0x0
   Pointer Size:  0
readelf: Warning: CU at offset 0 contains corrupt or unsupported version number: 0.
```

Could we install a newer version?